### ISSUE TITLE
Implement Link Test

### ISSUE THREAD

**Issue by P-Saha** [icgc-argo/e2e-test#32]
Add tests for broken links and checking where links are being directed to, and if they are going to the right pages. I.e. making sure links are redirecting to the proper pages.

**Comment by rosibaj** [comment]
@P-Saha @joneubank should this be included as part of the link fixing ticket?

**Comment by P-Saha** [comment]
@rosibaj I don't think it should be included as part of the link fixing ticket. This ticket will take some time due to needing to add ids to many links and buttons. It is not a very high priority and would probably be a bit too much right now. For scalability we should have ids required on all buttons and links so that tests can be made very easily in the future, along with a file for storing ids for reference.

**Comment by joneubank** [comment]
I've been discussing that change to the button component for a bit now. Definitely feels like a worthwhile ticket to update the component in order to streamline selenium test writing
